src/builders/dev-server/index.ts

```diff
@@ -24,6 +26,17 @@ export class SingleSpaDevServer extends DevServerBuilder {
         SINGLE-SPA-ANGULAR: Angular dev server is serving application as a single module`);
         return webpackConfig;
     }    
+    _buildServerConfig(root, projectRoot, options, browserOptions) {
+        // @ts-ignore
+        const devServerConfig = super._buildServerConfig(root, projectRoot, options, browserOptions);
+
+        return webpackMerge.smart(devServerConfig, {
+            // @ts-ignore
+            contentBase: path.resolve(root, projectRoot.serveDirectory || '../'),
```

Comment:
A directory that will serve static assets, including the [single-spa root config's html file](https://github.com/CanopyTax/single-spa-angular#check-if-it-works)